packages/basic-crawler/src/internals/basic-crawler.ts

```diff
@@ -587,8 +587,14 @@ export class BasicCrawler<Context extends CrawlingContext = BasicCrawlingContext
         this.autoscaledPoolOptions = { ...autoscaledPoolOptions, ...basicCrawlerAutoscaledPoolConfiguration };
     }
 
-    private getPeriodicLogger() {
+    private setStatusMessage(message: string, options: SetStatusMessageOptions = {}) {
+        this.log.info(`Setting${options.isStatusMessageTerminal ? ' terminal' : ''} status message: ${message}`);
```

Comment:
we could use `warning` level here when we log the message about errors.

being able to specify the log level was recently requested, so let's support that too (in actor sdk too)